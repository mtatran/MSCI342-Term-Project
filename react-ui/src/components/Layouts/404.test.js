import React from "react";
import { create } from "react-test-renderer";
import NotFound from './404';

describe('404 Snapshot', () => {
  test('snapshot renders', () => {
    const component = create(<NotFound />);
    expect(component.toJSON()).toMatchSnapshot();
  });

});