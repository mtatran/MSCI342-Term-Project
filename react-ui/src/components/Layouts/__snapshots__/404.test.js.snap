// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Snapshot snapshot renders 1`] = `
<h2>
  404 page not found 
</h2>
`;
